sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TypesClient.cs

```diff
@@ -0,0 +1,663 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.IoT.TimeSeriesInsights
+{
+    /// <summary>
+    /// Types client that can be used to perform operations for Time Series types.
```

Comment:
I would add something like

operations to CREATE, READ, UPDATE, and DELETE on Time Series Types and their associated variables. (however you want to keep the case for create/read etc)

The same comment applies to other clients.